exp-player/addon/components/exp-lookit-story-page/component.js

```diff
@@ -336,6 +347,16 @@ export default ExpFrameBaseUnsafeComponent.extend(FullScreen, VideoRecord,  {
         }
     },
 
+    parentTextStyle: Ember.computed('parentTextBlock', function() {
+        var parentTextBlock = this.get('parentTextBlock') || {};
+        var css = parentTextBlock.css || {};
+        var cssString = '';
+        $.each(css, function(propName, value) {
+            cssString += new Ember.Handlebars.SafeString(`${propName}:  ${value};`);
```

Comment:
I'm a bit nervous about this line; we generally try to avoid using SafeString.

The reason is that it totally trusts the content to be rendered exactly as HTML- you need to be sure that you 100% trust every person with access to any field that is part of the safestring, both now, and for any future possible use or adaptation of this component/code/template. Otherwise, they can insert arbitrary code of their own into every page you view (JS that sends API calls with your credentials, ad trackers, etc). 

The formal term is [XSS](https://www.owasp.org/index.php/Cross-site_Scripting_(XSS)), and this risk is the reason why most web frameworks escape all text by default. Is there a way to avoid this? For example, would [jquery css method](http://api.jquery.com/css/) meet your needs?